src/components/Books.js

```diff
@@ -1,32 +1,21 @@
+import { useSelector, useDispatch } from 'react-redux';
+import { addedBook } from '../redux/books/bookSlice';
 import FormInput from './InputBook';
 import BookList from './BookList';
 
 const Books = () => {
-  const booksArray = [
-    {
-      id: 1,
-      title: 'The Hunger Games',
-      author: 'Suzanne Collins',
-      completed: false,
-    },
-    {
-      id: 2,
-      title: 'Dune',
-      author: 'Frank Herbert',
-      completed: false,
-    },
-    {
-      id: 3,
-      title: 'Capital in the Twenty-First Century',
-      author: 'Suzanne Collins',
-      completed: false,
-    },
-  ];
+  const booksArray = useSelector((state) => state.books.numOfBooks);
+  const dispatch = useDispatch();
+  console.log(booksArray);
+  const handleUserInput = (title, author) => {
```

Comment:
- [ ] It would be nice if you remove the console.log because it shows important data in the console window of the browser which is not good for the security of your app 🤔 So it would be nice if you remove the console.log to follow the JavaScript best practices 👍